docs/app/partials/getting-started.tmpl.html

```diff
@@ -18,6 +18,22 @@
       <li><a href="https://github.com/angular/material#cdn">Using a CDN</a> (example below)</li>
     </ul>
 
+    <h3>Include Optional Dependencies</h3>
+    <p>
+      Angular Material integrates with some additional libraries which you may optionally include:
+    </p>
+    <ul style="margin-bottom: 2em;">
+      <li>
+        <a href="https://docs.angularjs.org/api/ngMessages">ngMessages</a>
+        - Provides a consistent mechanism for displaying form errors and other messages.
+      </li>
+
+      <li>
+        <a href="https://docs.angularjs.org/api/ngRoute">ngRoute</a>
+        - Provides a clean routing system for your application.
+      </li>
+    </ul>
+
```

Comment:
We should probably have 

* some verbiage about requirements for AngularJS 1.3.19 (or higher), or 1.4.7 (or higher), 1.5.x.
* an explicit notice regarding the dependency on **ngMaterial**:  `angular.module("<yourAppName>", ['ngMaterial'])`